packages/clerk-js/src/core/clerk.ts

```diff
@@ -2368,8 +2375,8 @@ export class Clerk implements ClerkInterface {
     this.#authService = await AuthCookieService.create(
       this,
       this.#fapiClient,
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      this.#instanceType!,
+
+      this.#instanceType,
```

Comment:
I have to check why my linter is making those changes on editor save... it ends up leading to a lot of build errors.